x-pack/plugins/ml/public/application/components/saved_objects_warning/saved_objects_warning.tsx

```diff
@@ -93,7 +93,7 @@ export const SavedObjectsWarning: FC<Props> = ({ jobType, onCloseFlyout, forceRe
         <>
           <FormattedMessage
             id="xpack.ml.jobsList.missingSavedObjectWarning.description"
-            defaultMessage="Some jobs are missing or have incomplete saved objects. "
+            defaultMessage="Some jobs or trained models are missing or have incomplete saved objects. "
```

Comment:
The text for the link should probably be changed too. `Synchronize your jobs and trained models` or `Sychronize objects`?

![image](https://user-images.githubusercontent.com/7405507/155368505-b74ebeea-4410-4d53-b93f-7b73b5ea24e0.png)